docs/docs/cryptography.md

```diff
@@ -138,19 +138,42 @@ Additionally, Lagrange interpolation also guarantees that no set of less than $t
 
 In summary, a threshold BLS signature $\sigma$ exhibits all properties required for publicly-verifiable, unbiased, unpredictable, and distributed randomness.
 
+#### Smaller signatures for resource constrained applications
+
+The above description of the signature threshold scheme accurately describes how we generate signatures that contain all the necessary properties for randomness under the `pedersen-bls-chained` and `pedersen-bls-unchained` schemes. 
+Under that scheme, messages are mapped to a point on $\mathbb{G}_1$ and public keys are a point on $\mathbb{G}_2$. As a result, message signatures are 96 bytes long and public keys are 48 bytes long. For systems where users are signing large messages and/or storing a lot of public keys, this trade-off makes a lot of sense: a 96 byte signature will be a small proportion of the entire message stored or transmitted. Additionally, signatures can be aggregated to save space.
+For drand however, we create a single public key at foundation of the network and emit random beacons (and thus signatures) at a constant rate as long as the network is running.
+As of [v1.5.0](https://github.com/drand/drand/releases/tag/v1.5.0-testnet) drand supports a new scheme, `bls-unchained-on-g1`, which exploits the bilinearity property of the BLS12-381 signature scheme to swap the generator groups for signatures and public keys such that signatures are created on $\mathbb{G}_2 and public keys are a point on $\mathbb{G}_1. This reduces the storage requirements of the node operators by 50% as well as reducing costs for any downstream users that require storing randomness beacons (such as smart contracts).
+
+For many applications using BLS12-381, messages are large and the signature constitutes a small part of the payload size. For public-facing applications, such as in blockchain ecosystems, users of the system store their public key on a
+Many applications built with BLS12-381 create many keypairs and subsequently publish or store many public keys.
+Thus for that scheme, the following operations apply:
```

Comment:
This reads gibberish. What did you mean?

Reply:
Errr think I wrote two versions and forgot to finish one haha

Reply:
okay, have removed it